### PR TITLE
1 html section 1

### PR DESCRIPTION
**Pull request by Evbgsl**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,25 @@
...
   <!-- Секция 1 -->
-  <section>Секция 1</section>
+  <section class="section" id="download" >
+    <div class="wrapper">
+      <div class="text-block">
+        <h1 class="title">Peaceful shopping mindful money</h1>
+        <p class="copy">See how we can help with making your shopping experience and money management more ZEN.</p>
+        <div class="links">
+          <a class="link" href="https://apple.com">
+            <img src="./assets/icons/apple.svg" alt="Apple icon">
```
Comment (ghost) [inline index.html]: Recommended to close the single tag
```
<img src="./assets/icons/apple.svg" alt="Apple icon" />
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# ng-free-zen
+
+
```
Comment (ghost) [inline README.md]: Delete the empty string

**Review by ghost — APPROVED** [review]
Done

File: index.html (hunk 1 above)
Comment (Evbgsl) [reply index.html]: ok

File: README.md (hunk 2 above)
Comment (Evbgsl) [reply README.md]: ok

**Review by ghost — APPROVED** [review]
Done

**Pull request merged**